### ISSUE TITLE
Switch Tabs

### ISSUE THREAD

**Issue by rsidhaarth** [Eugeny/terminus#6787]
**Describe the problem**:

Hi,

I am coming from iTerm. And I used to switch among tabs by using the command key + left and command key + right among the tabs.

I went to Tabby preference, hotkeys and set this.

![image](https://user-images.githubusercontent.com/16856248/179217412-ce8191c1-d565-4777-b102-53dcd06dde35.png)

when I try to switch from one ssh tab to other tab it is not working.

**Comment by Eugeny** [comment]
Please check whether it conflicts with other assignments, in particular "beginning of the line" uses Cmd-Left by default.

**Comment by Eugeny** [comment]
Good point! You're looking for the `next-tab` and `previous-tab` hotkeys.
